R/readTRACCS.R

```diff
@@ -0,0 +1,233 @@
+#' Read TRACCS road transportation data.
+#'
+#'
+#' @param subtype One of the possible subtypes, see default argument.
+#' @return magclass object
+#'
+#' @examples
+#' \dontrun{
+#' a <- readSource("TRACCS")
+#' }
+#' @author Alois Dirnaichner
+#' @seealso \code{\link{readSource}}
+#' @importFrom readxl read_xlsx
+#' @importFrom data.table as.data.table
+#' @importFrom magclass setComment
+#' @importFrom stringr str_extract
+readTRACCS <- function(subtype = c("roadFeDemand", "fuelEnDensity", "loadFactor", "annualMileage",
+                                   "roadVkmDemand", "roadTkmDemand", "roadPkmDemand", "railFeDemand")) {
```

Comment:
I really like listing all available subtypes as you did here, maybe you could do the same for all EDGE-T related functions with subtypes?

Reply:
sounds reasonable. As soon as we internally agree on the structure and the available subtypes we should do that.